composer.json

```diff
@@ -17,7 +17,12 @@
         "rebelcode/module-iterator-abstract": "dev-develop",
         "rebelcode/module-locator": "dev-develop",
         "rebelcode/module-config-interface": "dev-develop",
-        "rebelcode/module-config-validator": "dev-develop"
+        "rebelcode/module-config-validator": "dev-develop",
+        "rebelcode/module-factory-abstract": "dev-develop",
+        "dhii/data-key-value-aware-trait": "dev-task/initial-traits",
+        "dhii/callback-aware-trait": "dev-task/initial-traits",
+        "dhii/config-aware-trait": "dev-task/initial-traits",
```

Comment:
I would really much prefer these things to be embedded into this repo at this point. Namely, `callback-aware-trait` and `config-aware-trait`. The `data-key-value-aware-trait` can stay separately, because it corresponds to and extends the already existing structure.